robot/Cumulus/resources/locators_50.py

```diff
@@ -259,6 +259,9 @@
         'search_result': '//tbody/tr/td/a/span[contains(text(),"{}")]',
         'formula_txtarea': '//textarea[@id = "{}"]',
         'object_result': '//th/a[text()="{}"]',
+		'field_result': '//td/a/span[text()="{}"]',
```

Comment:
please adjust formatting

Reply:
This looks fine on the editor. Not sure why it is showing it that way here